baselines/deepq/deepq.py

```diff
@@ -306,8 +306,9 @@ def make_obs_ph(name):
                 # Update target network periodically.
                 update_target()
 
-            mean_100ep_reward = round(np.mean(episode_rewards[-101:-1]), 1)
             num_episodes = len(episode_rewards)
+            mean_100ep_reward = round(np.mean(episode_rewards[-101:-1]), 1) if \
+                num_episodes > 1 else 0.0
```

Comment:
so in this case the first reward would always be 0? I would argue that NaN is a better value - for instance, in the case when all real rewards are negative

Reply:
I agree and make this change.